tests/director/tests/formation_templates_test.go

```diff
@@ -0,0 +1,191 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/pkg/assertions"
+	"github.com/kyma-incubator/compass/tests/pkg/fixtures"
+	"github.com/kyma-incubator/compass/tests/pkg/testctx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var formationTemplateInput = graphql.FormationTemplateInput{
+	Name:                          "test-formation-template",
```

Comment:
extract the name as separate variable and use it in the tests for assertions and logs